CONTRIBUTING.md

```diff
@@ -151,3 +151,34 @@ Reviewers, please ensure that the CLA has been signed by referring to [the contr
 
 For changes that involve user facing copy please include `docs-cloud-team` as a reviewer.
 
+
+## Debugging in VSCode
```

Comment:
can we move this under 
<img width="307" alt="Screenshot 2023-03-10 at 13 49 10" src="https://user-images.githubusercontent.com/461027/224332753-a7b186a6-1c8d-4e80-8893-d95db42da404.png">
>Submitting Patches

not sure if at the same level of "Building and Testing" or under it